src/ResourceManager/Compute.ManagedService/Commands.Compute.ManagedService/Tools/ConvertTo-AzureRmVhd.ps1

```diff
@@ -0,0 +1,160 @@
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Convert Hyper-V VM to Azure supported virtual hard disk files
+#>
+function ConvertTo-AzureRmVhd
+{
+    [CmdletBinding()]
+    [OutputType([string[]])]
+    param
+    (
+        [Parameter(ParameterSetName = 'Hyper-V', Mandatory = $true)]
+        [string]$HyperVVMName,
+        
+        [Parameter(ParameterSetName = 'Hyper-V', Mandatory = $true)]
+        [string]$ExportPath,
```

Comment:
@huangpf should we make the default value of this parameter to be the current directory? That would eliminate the need for the user to provide `-ExportPath "."`

Reply:
In my opinion, it's better for users to explicitly declare the export path.

Reply:
This is OK for a preview, but the default should be the current directory.  Users may use -WhatIf if they are concerned about what the cmdlet would do

Reply:
ok...